javaparser-core/src/main/java/com/github/javaparser/printer/PrettyPrinterConfiguration.java

```diff
@@ -30,6 +30,7 @@
     private boolean printComments = true;
     private boolean printJavaDoc = true;
     private boolean columnAlignParameters = false;
+    private boolean columnAlignFirstMethodChain = false;
```

Comment:
I'm fine with all these options enabled by default - for configuration options we say "you get everything, if you don't want it turn it off." It will surprise some users, but I will put a clear comment in the release notes.